srcjs/shinyapp.js

```diff
@@ -487,14 +500,14 @@ var ShinyApp = function() {
     this.$updateConditionals();
   };
 
-
   // A function for sending messages to the appropriate handlers.
   // - msgObj: the object containing messages, with format {msgObj.foo, msObj.bar
   this._sendMessagesToHandlers = function(msgObj, handlers, handlerOrder) {
+    var tag, buffer, i;
```

Comment:
The `tag` and `buffer` vars don't appear to be used -- maybe this should just be left unchanged.